### ISSUE TITLE
[UI] [Enhancement] Remove remaining uneeded lang references

### ISSUE THREAD

**Issue by sparky8251** [jellyfin/jellyfin#388]
As part of #386 I also found a large number of unneeded lang file strings that are refereed to elsewhere.

The vast majority of these are still tied into JS that we dont need, but is referred to anyways. Since it requires additional effort, it has been left outside of the PR that removes the unreferenced strings.

Below is the complete list of strings I found that are referred to in code but we have no need for:

ButtonLearnMoreAboutJellyfinConnect 
ButtonSendInvitation 
ErrorAddingJellyfinConnectAccount1 
ErrorAddingJellyfinConnectAccount2 
ErrorAddingJellyfinConnectAccount3 
ErrorMessageEmailInUse 
ErrorRemovingJellyfinConnectAccount 
FeatureRequiresJellyfinPremiere 
GuestUserNotFound 
HeaderAddLocalUser 
HeaderBecomeProjectSupporter 
HeaderBranding 
HeaderBrandingHelp 
HeaderFreeApps 
HeaderInvitationSent
HeaderInvitations
HeaderInviteUser
HeaderInviteWithJellyfinConnect 
HeaderPendingInvitations 
HeaderSupportTheTeam 
HeaderSupporterBenefits 
HeaderTermsOfService 
HeaderWelcomeToJellyfin 
HowToConnectFromJellyfinApps 
Invitations 
InviteAnJellyfinConnectUser 
JellyfinIntroDownloadMessage 
JellyfinIntroDownloadMessageWithoutLink 
JellyfinIntroMessage 
LabelAddConnectSupporterHelp 
LabelConnectGuestUserName 
LabelConnectGuestUserNameHelp 
LabelConnectUserName 
LabelConnectUserNameHelp 
LabelEmail 
LabelEmailAddress 
LabelEnterConnectUserNameHelp 
LinkLearnMoreAboutSubscription 
MessageConnectAccountRequiredToInviteGuest 
MessageGuestSharingPermissionsHelp 
MessageJellyfinAccontRemoved 
MessageJellyfinAccountAdded 
MessagePendingJellyfinAccountAdded 
MessagePleaseAcceptTermsOfServiceBeforeContinuing 
MessagePluginRequiresSubscription 
MessagePremiumPluginRequiresMembership 
MessageProfileInfoSynced 
MessageThankYouForConnectSignUp 
MessageThankYouForConnectSignUpNoValidation 
MessageThankYouForSupporting 
MessageTrialExpired 
MessageTrialWillExpireIn 
RegisterWithPayPal 
TermsOfUseTermsOfUse 
TextEnjoyBonusFeatures 
XmlTvPremiere

**Comment by bfayers** [comment]
#394 Removes 

>XmlTvPremiere

(from html, not from strings)

**Comment by sparky8251** [comment]
Yes, hence why they are in this list. I've included this list with jellyfin specifically for the lang file cleanup, not the code.

**Comment by cvium** [comment]
It'd be nice if the JS files you're changing could be run through https://www.npmjs.com/package/deuglify and get cleaned up (warning: requires a bit of manual polish) in the same go. *Except emby-webcomponents and emby-apiclient*

**Comment by JustAMan** [comment]
Re de-uglifying and fixing in one go - this makes reviewing harder. If you do so please at least de-uglify in a separate commit (i.e. de-uglify, git commit, do the fix, git commit, submit a PR). Ideally those two things should go in separate PRs IMHO.
